lib/index.js

```diff
@@ -141,11 +141,11 @@ const open = (_args, opts = {}, extra = {}) => {
     if (platform === 'win32') {
       // spawnWithShell does not do the additional os.release() check, so we
       // have to force the shell here to make sure we treat WSL as windows.
-      options.shell = process.env.ComSpec
```

Comment:
setting options.shell was load bearing here, `spawnWithShell` needs that behavior.